src/Catalyst.Tools.KeyGenerator/Program.cs

```diff
@@ -96,9 +89,32 @@ private static IContainer LoadContainer()
                .WriteTo.Console()
                .CreateLogger()
                .ForContext(MethodBase.GetCurrentMethod().DeclaringType);
+            
             containerBuilder.RegisterInstance(_logger).As<ILogger>();
 
-            return containerBuilder.Build();
+            var container = containerBuilder.Build();
+
+            _userOutput = container.Resolve<IUserOutput>();
+            _commands = container.Resolve<IKeyGeneratorCommand[]>();
+            _peerSettings = container.Resolve<IPeerSettings>();
+
+            while (true)
+            {
+                try
+                {
+                    ParseCommand(args);
+                }
+                catch (Exception e)
+                {
+                    _logger.Error(e, nameof(ParseCommand));
+                }
+
+                // Non-interactive command
+                if (args.Length != 0)
+                {
+                    break;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Refactor the code in order to remove this break statement.](https://app.codacy.com/app/catalyst-network/Catalyst.Node/pullRequest?prid=4355202)